adminapi/src/main/java/io/minio/admin/DataUsageInfo.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage,
+ * (C) 2022 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio.admin;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+/** Represents Data usage of the current object APi. */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DataUsageInfo {
```

Comment:
Add all missing objects and make it complete than partial support.

Reply:
@balamurugana Done. As there are few model classes instead of just those two small ones, I've created new package to hold these. I've also included simple test that the full object can actually be deserialized via Jackson